### ISSUE TITLE
AIX build process and bash side-effects when using `process substitution`

### ISSUE THREAD

**Issue by aixtools** [AdoptOpenJDK/openjdk-build#2536]
**What are you trying to do?**
Standard build (on AIX) - manual or automated

**Expected behaviour:**
Build completes without 7500-8000 left over special files in /tmp
**Observed behaviour:**
Build completes (success or fail, not relevant, except that success has most leftover files):
roughly 7800 /tmp/sh-np* files>
e.g: see https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/2029 for a proposed *treatment* - a cronjob to daily scan /tmp for stale (jenkins) files left-over from the build process

Also see: https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/2029#issuecomment-802706761 for an explanation of what is happening - and perhaps a solution
**Any other comments:**

* While the current build-process is not directly at fault - it does suffer from a bash-bug. The bug is known, but not resolved.
* The way *process substitution* is being used in most, if not all, cases might be resolved by using traditional pipes and output re-direction.
* In short: *healing* might be much more effective than *treatment*

**Comment by aixtools** [comment]
Basically, why use something such as:
```
/usr/bin/printf "Building targets 'product-images legacy-jre-image test-image' in configuration 'aix-ppc64-normal-server-release'\n" >
>(/usr/bin/tee -a /home/aixtools/openjdk-build/build-farm/workspace/build/src/build/aix-ppc64-normal-server-release/build.log) 2> >(/usr/bin/tee -a /home/aixtools/openjdk-build/build-farm/workspace/build/sr
c/build/aix-ppc64-normal-server-release/build.log >&2)
```
rather than:
```
/usr/bin/printf "Building targets 'product-images legacy-jre-image test-image' in configuration 'aix-ppc64-normal-server-release'\n" 2>&1 | tee -a /usr/bin/tee -a /home/aixtools/openjdk-build/build-farm/workspace/build/sr
c/build/aix-ppc64-normal-server-release/build.log
```
